docs/compiler.md

```diff
@@ -181,6 +181,64 @@ Deployment works via an RPC server, an address of which is passed via `-r`
 option and should be signed using a wallet from `-w` option. More details can
 be found in `deploy` command help.
 
+#### Config file
+Configuration file contains following options:
+
+| Parameter | Description | Example |
+| --- | --- | --- |
+| `name` | Contract name in the manifest. | `"My awesome contract"`
+| `safemethods` | List of methods which don't change contract state | `["balanceOf", "decimals"]`
```

Comment:
s/`state`/`state.`

Reply:
, don't emit notifications and are available for anyone to call.